yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/object_type_mapping.psql

```diff
@@ -5,6 +5,7 @@ SELECT
     CASE 
         WHEN c.relkind = 'r' THEN 'table' 
         WHEN c.relkind = 'p' THEN 'partitioned table'
+        WHEN (c.relkind = 'i' AND pi.indisprimary) THEN 'primary key'
```

Comment:
nit: This might be a pain to change :) but, `primary key index` just to be explicit?

Reply:
ah yeah it will be big change. I believe we can live with `primary key` ? 
TableIndexStats have IsIndex as true for PKs